modules/apps/29-fee/ibc_middleware_test.go

```diff
@@ -845,20 +868,43 @@ func (suite *FeeTestSuite) TestOnTimeoutPacket() {
 		{
 			"success",
```

Comment:
Also for explicitness:

```suggestion
			"success: no refund",
```